src/LearningOutcomes/LearningOutcomeRouteHandler.spec.ts

```diff
@@ -86,24 +45,28 @@ describe('POST /learning-objects/:id/learning-outcomes', () => {
 });
 
 describe('GET /learning-objects/:id/learning-outcomes/:outcomeId', () => {
-
-  it('should return a status of 200 and a Learning Outcome', (done) => {
-    request.get('/learning-objects/someObjectId/learning-outcomes/someLearningOutcomeId')
-    .expect('Content-Type', /json/)
-    .expect(200)
-    .then(res => {
-      expect(() => {
-        const _ = new LearningOutcome(res.body);
-      }).not.toThrow();
-      done();
-    });
+  it('should return a status of 200 and a Learning Outcome', done => {
```

Comment:
The test definition doesn't seem very specific as to what is really being asserted. What about something like this?

```suggestion
  it('should return a response body that can be converted to a valid Learning Outcome', done => {
```